src/Symfony/Installer/NewCommand.php

```diff
@@ -133,7 +139,22 @@ private function checkSymfonyVersionIsInstallable()
     {
         // 'latest' is a special version name that refers to the latest stable version
         // available at the moment of installing Symfony
-        if ('latest' === $this->version) {
+        if (self::VERSION_LATEST === $this->version) {
+            return $this;
+        }
+
+        // Get the Symfony versions list (http://symfony.com/versions.json)
+        $versionsList = $this->getVersionsList();
```

Comment:
Unless `symfony.org` plans to propose a proper "standard" download link for the LTS version, this behavior seems correct to me.